app/controllers/verify/step_controller.rb

```diff
@@ -8,5 +8,28 @@ class StepController < ApplicationController
     before_action :confirm_idv_attempts_allowed
 
     helper_method :step
+
+    private
+
+    def step_name
+      raise NotImplementedError, 'must implement step_name method'
+    end
```

Comment:
Ah, just noticed this was already merged. I will open an issue to refactor this later.